docs/bin/plugin_formatter.py

```diff
@@ -338,6 +342,7 @@ def get_plugin_info(module_dir, limit_to=None, verbose=False):
                                'returndocs': returndocs,
                                'categories': module_categories,
                                'primary_category': primary_category,
+                               'sub_category': sub_category
```

Comment:
It's generally a good idea to put a `,` at the end of the last element in a list or dict. Python lets you do this and it can prevent syntax errors in the future when rearranging the lines or adding new items.